build.d/80-dependencies

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# Copyright 2017 LasLabs Inc.
+
+import os
+import subprocess
+
+
+DIR_CUSTOM = '/opt/odoo/custom/dependencies'
+
+REQ_TYPES = {
+    'apt': 'apt-get install -y',
+    'pip': 'pip install',
+    'npm': 'npm install -g',
+    'gem': 'gem install',
+}
+
+
+def do_install(command, requirements_file):
+    with open(requirements_file, 'r') as fh:
+        requirements = fh.read().strip()
+    command += ' %s' % requirements
```

Comment:
Yikes, this loop is interesting, but I feel each install method has its things.

For instance,

- `apt` needs an `apt-get update` before installing the build deps, and needs `apt-get -y autoremove && rm -Rf /var/lib/apt/lists/*` after installing run deps.
- `pip` can better use its `--requirements` flag instead of this read-and-append hack. (maybe not required, see next point).
- the rest of methods would benefit of removing lines from its requirements file that start by `#`, so we allow comments there (for pip this would mean the same as the point above).

`rm /tmp/*` would be great too at the end of everything.